### DISCUSSION TITLE
Autogenerated Swift headers

### DISCUSSION THREAD

**Discussion by VitaliyGaliy** [callstack/react-native-builder-bob#393] [Q&A]
I am writing a RN library with the code in Swift. It has initialization part which I need to invoke from main application AppDelegate.m. Because of AppDelegate is written in Objective-C I need <moduleName>-Swift.h file. How can I generate one inside react-native-builder-bob library?

**Comment by atlj** [comment]
Hey, the Swift bridging header should be generated when you generate a Swift project. If you want to make your swift classes visible on objective-c, you can take a look at this document: https://developer.apple.com/documentation/swift/importing-swift-into-objective-c#Include-Swift-Classes-in-Objective-C-Headers-Using-Forward-Declarations
